README.md

```diff
@@ -7,3 +7,7 @@
 1. Install this module in the python environment for studio and lms.
 2. Add `"vectordraw"` to the Advanced Module List array in Advanced
    Settings in Studio for each course this should be installed in.
+
+## Testing
+
+1. For running the tests on Python 3.6 and Django versions 2.2 and 3.2 use the command `tox`
```

Comment:
Nit: I don't think you need to specify the versions here, "For running the tests use the command `tox`" should be clear enough.